addons/editor-theme3/forums/base.css

```diff
@@ -137,7 +141,11 @@
 .sb-custom-arg ~ .sb-label,
 .sb3-custom-arg,
 .sb3-custom-arg-dark,
-.sb3-custom-arg ~ .sb3-label {
+.sb3-custom-arg ~ .sb3-label,
+.sb-outline,
+.sb-outline ~ .sb-label,
+.sb-label:only-of-type:not(:last-child),
```

Comment:
This also matches the labels of some other blocks:
![image](https://github.com/ScratchAddons/ScratchAddons/assets/51849865/d381ad8e-04a6-41a6-81a2-b372a316bac8)
(blocks from [this post](https://scratch.mit.edu/discuss/post/7535381/))

Reply:
Fixed - looks like this now, which is still incorrect, but that's the same behavior as SA has without this PR

![image](https://github.com/user-attachments/assets/f57e203c-5e2b-4e43-85e3-2ef223b3d52f)